custom_model_runner/datarobot_drum/resource/transform_helpers.py

```diff
@@ -93,3 +93,13 @@ def parse_multi_part_response(response):
         parsed_response.update({key: value})
 
     return parsed_response
+
+
+def read_x_data_from_response(response):
+    reader = {
+        "arrow": read_arrow_payload,
+        "sparse": lambda d, k: pd.DataFrame.sparse.from_spmatrix(read_mtx_payload(d, k)),
```

Comment:
I'd rather see an internal function instead of a lambda. It's more readable that way, but this is fine.